src/components/TabList.tsx

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Div, DivProps, Flex, FlexProps } from 'honorable'
+import { Item } from '@react-stately/collections'
+import { useTab, useTabList, useTabPanel } from '@react-aria/tabs'
+import { TabListState } from '@react-stately/tabs'
+import { ItemProps, Node } from '@react-types/shared'
+import {
+  ComponentPropsWithRef,
+  HTMLAttributes,
+  RefObject,
+  useRef,
+} from 'react'
+
+import Tab from './Tab'
+
+type Renderer = (
+  props: HTMLAttributes<HTMLElement>,
+  ref: RefObject<any>,
```

Comment:
In general you should be able to use `unknown` instead of `any` in most places. It is type safe and does not trigger warnings. Everything can be assigned to `unknown` but `unknown` cannot be assigned to anything.

Reply:
I've updated most `any`s to other types, but I need `RefObject<any>` in this case or it causes all kinds of errors in usage.